react/wbooks/src/app/components/Login/index.js

```diff
@@ -0,0 +1,80 @@
+import React, { Component } from 'react';
+import { get, isArray } from 'lodash';
```

Comment:
why are you using lodash? we avoid using external libraries to get done easy jobs.

Reply:
Why not use lodash? I can take advantage of everything it already provides and not do it again. Also I know this code is extremely well tested